app/src/main/java/com/xmartlabs/template/ui/BaseFragment.java

```diff
@@ -0,0 +1,106 @@
+package com.xmartlabs.template.ui;
+
+import android.app.Activity;
+import android.app.ProgressDialog;
+import android.os.Bundle;
+import android.support.annotation.LayoutRes;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.v7.app.AlertDialog;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.hannesdorfmann.fragmentargs.FragmentArgs;
+import com.trello.rxlifecycle.components.support.RxFragment;
+import com.xmartlabs.template.BaseProjectApplication;
+import com.xmartlabs.template.R;
+import com.xmartlabs.template.ui.mvp.BaseMvpAppCompatActivity;
+
+import butterknife.ButterKnife;
+import butterknife.Unbinder;
+
+/**
+ * Created by diegomedina24 on 20/3/17.
+ */
+public abstract class BaseFragment extends RxFragment {
+  private Unbinder unbinder;
+  @Nullable
+  private ProgressDialog progressDialog;
```

Comment:
It is, although i don't think it should nor that we should use it here.
Rather use Android version here, as it stands the same.